src/test/java/com/stripe/functional/StripeResponseTest.java

```diff
@@ -25,7 +26,7 @@ public void testResponseIncluded() throws StripeException {
     final RequestOptions requestOptions = RequestOptions.builder()
         .setIdempotencyKey(idempotencyKey)
         .build();
-    final Customer customer = Customer.create(null, requestOptions);
+    final Customer customer = Customer.create((Map<String, Object>) null, requestOptions);
```

Comment:
Compiler complains that null now can imply both null for untyped param Map or typed `ApiRequestParams`. Existing integration with null map will likely break loudly with complier checks.